accumulo/values.yaml

```diff
@@ -72,6 +77,14 @@ accumulo:
     nodeSelector: {}
     tolerations: []
     affinity: {}
+  scanServers:
+    resourceGroups:
+    - resourceGroupName: group1
```

Comment:
You may already know this, but just in case I wanted to point out that you can use scan servers by just starting some and setting the consistency level on the scanner. The default resource group is named "default" (https://github.com/apache/accumulo/blob/2.1/core/src/main/java/org/apache/accumulo/core/spi/scan/ScanServerSelector.java#L46) and the default client configuration uses that value. If you are going to use non-default resource group names, then you will have to make client property modifications to use them.

Reply:
Feel free to ignore this comment, I made this comment before looking at your setup in site-config.yaml.